engine/runtime/src/test/java/org/enso/interpreter/test/ValuesGenerator.java

```diff
@@ -0,0 +1,202 @@
+package org.enso.interpreter.test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Value;
+import org.junit.Assert;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * The purpose of this class is to generate various values needed for other
+ * tests. Users of this support class can configure what kind of values they
+ * need - Enso, Java, JavaScript, etc. and then call appropriate methods to
+ * obtain such values. It's up to the tests to use these values meaningfully.
+ */
+class ValuesGenerator {
+  private final Context ctx;
+  private final Set<Language> languages;
+  private final Map<String, Value> values = new HashMap<>();
+
+  private ValuesGenerator(Context ctx, Set<Language> languages) {
+    this.ctx = ctx;
+    this.languages = languages;
+  }
+
+
+  public static ValuesGenerator create(Context ctx, Language... langs) {
+    var set = langs == null || langs.length == 0 ? EnumSet.allOf(Language.class)
+      : EnumSet.copyOf(Arrays.asList(langs));
+    return new ValuesGenerator(ctx, set);
+  }
+
+  private Value v(String k, String t, String s) {
+    var v = values.get(k);
+    if (v == null) {
+      var f = ctx.eval("enso", t + "\nn = " + s);
+      v = f.invokeMember("eval_expression", "n");
```

Comment:
```suggestion
      var f = ctx.eval(LanguageInfo.ID, t + "\nn = " + s);
      v = f.invokeMember(MethodNames.Module.EVAL_EXPRESSION, "n");
```

Reply:
It'd be better to completely rework this and use more standard way to access module members than invocation of a synthetic member.